Proposals/Glyph_Extension_Axis/ProposalSummary.md

```diff
@@ -0,0 +1,69 @@
+# Proposal Summary Form
```

Comment:
Let's make this title more specific:

#Proposal Summary Form: Glyph Extension

Reply:
Done